test/image/mocks/image_source_axis_reverse.json

```diff
@@ -0,0 +1,20 @@
+{"data": [
```

Comment:
We need to generate a baseline for this new mock. See https://github.com/plotly/plotly.js/blob/master/test/image/README.md#c-generate-or-update-existing-baseline-image
Please let me know if you had difficulty with docker installation so that I generate the baseline for you and simply push it to a dev branch.

Also we should add it and validate it by adding it here
https://github.com/plotly/plotly.js/blob/cdd836ccd709e3a871674f915589aeda7c6d501e/test/jasmine/tests/mock_test.js#L650
and here
https://github.com/plotly/plotly.js/blob/cdd836ccd709e3a871674f915589aeda7c6d501e/test/jasmine/tests/mock_test.js#L1723

Reply:
On second thoughts, because the image export is not using fast rendering, it doesn't make sense to add this into our image test.

I would however like to see a jasmine test that checks that the hover label is correct after reversing the axis. This test should be added in this `describe` block:
https://github.com/plotly/plotly.js/blob/cdd836ccd709e3a871674f915589aeda7c6d501e/test/jasmine/tests/image_test.js#L635-L673

Reply:
I added a Jasmine test to validate the value reported on hover. Interestingly, this works for the case when the y-axis is reversed, but fails when the x-axis is reversed. I checked, and the reported (wrong) value is indeed the value at the mirrored location. However, manually inspecting the image via the test dashboard *does* give the correct results when the x-axis is flipped. Can it be that in the Jasmine test a more direct method is called, which bypasses some logic that deals with reversed axii?  (I hope this all makes sense)

Reply:
> On second thoughts, because the image export is not using fast rendering, it doesn't actually test anything introduced in this PR so I would remove it from the final PR.

I am a bit confused. Do you mean we do not need a baseline image?

Reply:
You could safely remove this new mock `test/image/mocks/image_source_axis_reverse.json` which is unused.

Reply:
I removed the Mock. I put the code in a little html doc for testing instead: https://gist.github.com/almarklein/db9e1a2faec6890d1156e6ec546d7ecc

Reply:
I suggest you simply add a codepen to the PR description instead.